packages/ag-charts-community/src/chart/series/cartesian/lineUtil.ts

```diff
@@ -17,7 +17,7 @@ type PathPoint = {
     from?: { x: number; y: number };
     to?: { x: number; y: number };
     marker: MarkerChange;
-    moveTo: true | false | 'in' | 'out';
+    moveTo: 'yes' | 'no' | 'in' | 'out';
```

Comment:
`yes` | `no` | `maybe`

Reply:
Discussed offline, 2 to 1 in favour of `boolean` here rather than `yes` | `no`.